rbx_reflection_database/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # rbx\_reflection_database Changelog
 
 ## Unreleased Changes
+- The database may now be loaded dynamically from the local file system. ([#376])
+  The location is OS-dependent but it will only be loaded if one exists. The location may also be manually specified using the `RBX_DATABASE` environment variable.
+
+  `get` is unchanged in use, but will return a locally stored database if it exists, and the bundled one if not. Two new methods were added: `get_bundled` will only fetch the local database and `get_local` will only fetch a locally stored one.
```

Comment:
This is not accurate to the new implementation, because rbx_reflection_database::get now returns a Result. That's a breaking change, which should be pointed out.